packages/flutter_tools/pubspec.yaml

```diff
@@ -132,6 +132,11 @@ dev_dependencies:
   node_preamble: 1.4.7 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
   shelf_packages_handler: 1.0.4 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
 
+dependency_overrides:
+  mustache:
+    git:
+      url: git@github.com:jonahwilliams/mustache.git
```

Comment:
This will have to be https for CI.  e.g. `https://github.com/jonahwilliams/mustache.git`